app/components/form-input.hbs

```diff
@@ -1,6 +1,7 @@
 <div class="field">
   <label for={{@for}}>{{@label}}:</label>
-  <Input
+  <div class="field">
+<Input
```

Comment:
Do we need this div `field` for each input, can we put a separate if statement which check if the field is location, and then renders a different form input component which is inside a div and contains a button

Reply:
@vinayak-trivedi If we are doing this it is breaking whole form css of the form